main/WebServer.cpp

```diff
@@ -10560,6 +10568,20 @@ namespace http
 						root["result"][ii]["SetPoint"] = szTmp;
 						root["result"][ii]["HaveTimeout"] = bHaveTimeout;
 						root["result"][ii]["TypeImg"] = "override_mini";
+
+						if (_hardwareNames[hardwareID].HardwareTypeVal == HTYPE_VirtualThermostat)
+						{
+							TOptionMap optionsMap = m_sql.BuildDeviceOptions(sOptions);
+							//build option values
+							for (const auto& itt : optionsMap)
+							{
+								std::string optionName = itt.first.c_str();
+								std::string optionValue = itt.second;
+								root["result"][ii][optionName.c_str()] = optionValue.c_str();
```

Comment:
please remove the .c_str() on both places

Reply:
Ok done.